backend/src/apiserver/main.go

```diff
@@ -85,6 +84,17 @@ func main() {
 		template.Launcher = common.GetStringConfig(launcherEnv)
 	}
 
+	backgroundCtx, backgroundCancel := context.WithCancel(context.Background())
```

Comment:
This is important to keep so that the context is cancelled if there is a signal for the process to stop.
```suggestion
	backgroundCtx, backgroundCancel := context.WithCancel(signals.SetupSignalHandler())
```